packages/client/src/generation/TSClient/PrismaClient.ts

```diff
@@ -106,8 +48,8 @@ function clientTypeMapOthersDefinition(this: PrismaClientClass) {
   })
 
   const argsResultMap = {
-    $executeRaw: { args: 'Prisma.Sql', result: 'any' },
-    $queryRaw: { args: 'Prisma.Sql', result: 'any' },
+    $executeRaw: { args: '[query: TemplateStringsArray | Prisma.Sql, ...values: any[]]', result: 'any' },
+    $queryRaw: { args: '[query: TemplateStringsArray | Prisma.Sql, ...values: any[]]', result: 'any' },
```

Comment:
This looks like a revert of what was done in https://github.com/prisma/prisma/pull/19787
Is that expected?

Reply:
Yes, I needed to restore it so I can type regular calls as well. The `query` extensions have a spacial case instead.